Source/WebCore/rendering/RenderObject.h

```diff
@@ -467,7 +467,7 @@ class RenderObject : public CachedImageClient, public CanMakeCheckedPtr {
     bool isLegacySVGForeignObject() const { return type() == Type::LegacySVGForeignObject; }
     bool isSVGForeignObject() const { return type() == Type::SVGForeignObject; }
     virtual bool isLegacySVGResourceContainer() const { return false; }
-    virtual bool isSVGResourceContainer() const { return false; }
+    bool isSVGResourceContainer() const { return type() == Type::SVGResourceClipper; }
```

Comment:
So when we add more LBSE specific resources that inherit from RenderSVGResourceContainer, we just add them as additional check here, right?

Reply:
Correct. And if it becomes too crazy, feel free to go back to a virtual function.